src/Discord.Net.Core/CDN.cs

```diff
@@ -13,6 +13,15 @@ public static string GetUserAvatarUrl(ulong userId, string avatarId, ushort size
             string extension = FormatToExtension(format, avatarId);
             return $"{DiscordConfig.CDNUrl}avatars/{userId}/{avatarId}.{extension}?size={size}";
         }
+        public static string GetDefaultUserAvatarUrl(string discriminator)
+        {
+            if (string.IsNullOrEmpty(discriminator))
+                throw new ArgumentNullException(nameof(discriminator));
+
+            int discriminator_mod = Convert.ToInt32(discriminator) % 5;
```

Comment:
We already know the discriminator is a ushort, you may as well just use `ushort.Parse` here.